start-site/src/main/resources/application.yml

```diff
@@ -1105,19 +1115,27 @@ initializr:
               description: Using Spring Cloud Gateway
         - name: Ribbon [Maintenance]
           id: cloud-ribbon
-          description: Client-side load-balancing with Spring Cloud Netflix and Ribbon.
+          description: "Client-side load-balancing with Spring Cloud Netflix and Ribbon. Maintenance mode, suggested replacement: Spring Cloud LoadBalancer."
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-netflix-ribbon
           compatibilityRange: "[2.0.0.RELEASE,2.4.0-M1)"
           links:
             - rel: guide
               href: https://spring.io/guides/gs/client-side-load-balancing/
               description: Client-side load-balancing with Ribbon and Spring Cloud
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-netflix/docs/2.2.6.RELEASE/reference/html/#spring-cloud-ribbon
```

Comment:
Isn't there a chance that this link will go out of sync? It's tied to a particular version.

Reply:
This module is no longer available in `3.0.0`, so we can't use `current`.